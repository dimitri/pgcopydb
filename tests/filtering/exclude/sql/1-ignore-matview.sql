select schemaname, matviewname, ispopulated from pg_matviews order by 1, 2;
