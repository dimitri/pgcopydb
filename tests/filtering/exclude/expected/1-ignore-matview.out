-[ RECORD 1 ]-----------------------
schemaname  | foo
matviewname | matview_1
ispopulated | t
-[ RECORD 2 ]-----------------------
schemaname  | foo
matviewname | matview_1_exclude_data
ispopulated | f

